tests/richText.js

```diff
@@ -553,9 +553,13 @@ define([
                 it("cursor should be at end of input on focus", function () {
                     var editor = widget.input.editor;
                     widget.setValue('testing cursor');
-                    editor.focus();
-                    var selection = editor.getSelection(true);
-                    assert.strictEqual(selection.getNative().focusOffset, 14);
+                    // Make sure focus is elsewhere, then focus on the rich text input
+                    editor.on('instanceReady', function() {
+                        $('[name=property-nodeID]').focus();
+                        editor.focus();
+                        var selection = editor.getSelection(true);
+                        assert.strictEqual(selection.getNative().focusOffset, 14);
```

Comment:
Looks like the end of the editor (This was one of my original tests). I think ckeditor has some magic constants that can be used for end of input instead of having this number